process/peer/interceptedValidatorInfo_test.go

```diff
@@ -0,0 +1,221 @@
+			GetValidatorWithPublicKeyCalled: func(publicKey []byte) (validator nodesCoordinator.Validator, shardId uint32, err error) {
+				return nil, 0, expectedErr
+			},
+		}
+
+		ivi, _ := NewInterceptedValidatorInfo(args)
+		require.False(t, check.IfNil(ivi))
+
+		assert.Equal(t, expectedErr, ivi.CheckValidity())
+	})
+	t.Run("should work", func(t *testing.T) {
+		t.Parallel()
+
+		args := createMockArgInterceptedValidatorInfo()
+		args.NodesCoordinator = &shardingMocks.NodesCoordinatorStub{
+			GetValidatorWithPublicKeyCalled: func(publicKey []byte) (validator nodesCoordinator.Validator, shardId uint32, err error) {
+				return nil, 0, nil
+			},
+		}
+
+		ivi, _ := NewInterceptedValidatorInfo(args)
+		require.False(t, check.IfNil(ivi))
+
+		assert.Nil(t, ivi.CheckValidity())
+	})
+}
+
+func testInterceptedValidatorInfoPropertyLen(property string, tooLong bool) func(t *testing.T) {
+	return func(t *testing.T) {
+		t.Parallel()
+
+		value := []byte("")
+		expectedError := process.ErrPropertyTooShort
+		if tooLong {
+			value = make([]byte, 130)
+			expectedError = process.ErrPropertyTooLong
+		}
+
+		args := createMockArgInterceptedValidatorInfo()
+		ivi, _ := NewInterceptedValidatorInfo(args)
+		require.False(t, check.IfNil(ivi))
+
+		switch property {
+		case publicKeyProperty:
+			ivi.validatorInfo.PublicKey = value
+		case listProperty:
+			ivi.validatorInfo.List = string(value)
+		case rewardAddressProperty:
+			ivi.validatorInfo.RewardAddress = value
+		default:
+			assert.True(t, false)
+		}
+
+		err := ivi.CheckValidity()
+		assert.True(t, strings.Contains(err.Error(), expectedError.Error()))
+	}
+}
+
+func TestInterceptedValidatorInfo_Getters(t *testing.T) {
+	t.Parallel()
+
+	args := createMockArgInterceptedValidatorInfo()
+	ivi, _ := NewInterceptedValidatorInfo(args)
+	require.False(t, check.IfNil(ivi))
+
+	validatorInfo := createMockValidatorInfo()
+	validatorInfoBuff, _ := args.Marshalizer.Marshal(validatorInfo)
+	hash := args.Hasher.Compute(string(validatorInfoBuff))
+
+	assert.True(t, ivi.IsForCurrentShard())
+	assert.Equal(t, *validatorInfo, ivi.ValidatorInfo())
+	assert.Equal(t, hash, ivi.Hash())
+	assert.Equal(t, interceptedValidatorInfoType, ivi.Type())
+
+	identifiers := ivi.Identifiers()
+	assert.Equal(t, 1, len(identifiers))
+	assert.Equal(t, hash, identifiers[0])
+
+	str := ivi.String()
+	assert.True(t, strings.Contains(str, fmt.Sprintf("pk=%s", logger.DisplayByteSlice(ivi.validatorInfo.PublicKey))))
+	assert.True(t, strings.Contains(str, fmt.Sprintf("shard=%d", validatorInfo.ShardId)))
+	assert.True(t, strings.Contains(str, fmt.Sprintf("list=%s", validatorInfo.List)))
+	assert.True(t, strings.Contains(str, fmt.Sprintf("index=%d", validatorInfo.Index)))
+	assert.True(t, strings.Contains(str, fmt.Sprintf("tempRating=%d", validatorInfo.TempRating)))
+	assert.True(t, strings.Contains(str, fmt.Sprintf("rating=%d", validatorInfo.Rating)))
+}
+
+func TestInterceptedValidatorInfo_Identifiers(t *testing.T) {
```

Comment:
empty tests?

Reply:
forgot to delete the lines as all getters are tested in `TestInterceptedValidatorInfo_Getters`